test/sql/restricted_placeholders.sql

```diff
@@ -8,3 +8,10 @@ select set_config('response.headers', '[{"X-Special-Header": "a-value"}]', true)
 
 -- placeholder restricted values fail on a SET
 set another.placeholder to 'special-value';
+\echo
+
+-- doesn't crash after a show all
+\o /dev/null
+show all;
+\o
+select 1;
```

Comment:
The `SHOW ALL` output is big - this hides it from the test output but still confirms that pg doesn't crash.